src/lib/mina.ts

```diff
@@ -475,10 +475,14 @@ function LocalBlockchain({
         }
       });
       return {
-        wait: async () => {},
+        wait: async (options?: { maxAttempts?: number; interval?: number }) => {
```

Comment:
Can you rename `options` to `_options`, to get rid of the unused variable warning? :D